src/Optional.js

```diff
@@ -53,13 +57,19 @@ export const optional = curry((preview, set) => new Optional(preview, set))
 // optionalProp : String → Optional s a
 export const optionalProp = (key) =>
   optional(
-    (obj) => obj[key] ?? notFound,
+    (obj) => {
+      if (key in obj) return obj[key]
+      else return notFound
+    },
     (val, obj) => (obj[key] ? { ...obj, [key]: val } : obj),
   )
 
 // optionalIx : Number → Optional s a
 export const optionalIx = (index) =>
   optional(
-    (obj) => obj[index] ?? notFound,
```

Comment:
Why this change? It looked better before 😢

Reply:
It turns out that some of the tools I use don't understand `??` yet. While doing so I also discovered a bug, because this won't work if `obj` was undefined.